### ISSUE TITLE
Swagger Documentation Correction

### ISSUE THREAD

**Issue by aderrose** [mailcow/mailcow-dockerized#3696]
**Prior to placing the issue, please check following:** *(fill out each checkbox with an `X` once done)*
- [X] I understand, that not following or deleting the below instructions, will result in immediate closing and deletion of my issue.
- [X] I have understood that answers are voluntary and community-driven, and not commercial support.
- [X] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

**Description of the bug**:

In the Swagger API documentation for the Delete Alias API command the example is given as
```JSON
[
  [
    "6",
    "9"
  ]
]
```
which doesn't work and generates the following error
```JSON
[
  {
    "type": "danger",
    "log": [
      "mailbox",
      "delete",
      "alias",
      {
        "id": [
          [
            "110"
          ]
        ]
      },
      null
    ],
    "msg": "access_denied"
  }
]
```
Having compared the example to the older apiary documentation it looks like it should be
```JSON
[
  "6",
  "9"
]
```
with just the one set of JSON array brackets and this works when sent via Swagger or Postman.

**Docker container logs of affected containers**:

**Reproduction of said bug**:

Bug can be reproduced by sending the incorrect JSON to the /api/v1/delete/alias endpoint.

**System information**:

| Question | Answer |
| --- | --- |
| My operating system | Debian 10 |
| Is Apparmor, SELinux or similar active? | Not sure |
| Virtualization technlogy (KVM, VMware, Xen, etc - **LXC and OpenVZ are not supported** | Not sure, I use a Linode VM |
| Server/VM specifications (Memory, CPU Cores) | 2 cores, 4 GB RAM |
| Docker Version (`docker version`) | 19.03.12 |
| Docker-Compose Version (`docker-compose version`) | 1.26.2 |
| Reverse proxy (custom solution) | None used |

- Output of `git diff origin/master`, any other changes to the code? If so, **please post them**.
- All third-party firewalls and custom iptables rules are unsupported. *Please check the Docker docs about how to use Docker with your own ruleset*. Nevertheless, iptabels output can help us to help you: `iptables -L -vn`, `ip6tables -L -vn`, `iptables -L -vn -t nat` and `ip6tables -L -vn -t nat`.
- DNS problems? Please run `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and post the output.

**Comment by ntimo** [comment]
Will check why the docs are wrong there 🤔. And fix it later today.

**Comment by aderrose** [comment]
I've also noticed another small correction that is needed.

The Edit Alias example shows the last line as a string
```JSON
"items": "6"
```
where it needs to be an array of string
```JSON
"items": [ "6" ]
```
I've tested this with Postman and the string fails with a blank body returned and the array of string works returning a populated body as expected.

There are a couple of small spelling mistakes as well.
1. The Logs header has sysmte rather than system
2. The Queue Manager header has mailque rather than mail queue
3. The Address Rewriting header has BBC maps rather than BCC maps
